api/src/functionalTest/java/uk/gov/hmcts/reform/divorce/caseapi/SolicitorCreateAboutToStartTest.java

```diff
@@ -21,14 +24,24 @@
 import static uk.gov.hmcts.reform.divorce.ccd.event.solicitor.SolicitorCreate.SOLICITOR_CREATE;
 
 @SpringBootTest
-public class DefaultLanguagePreferenceSettingTest extends FunctionalTestSuite {
+public class SolicitorCreateAboutToStartTest extends FunctionalTestSuite {
     private static final String SOLICITOR_CREATE_ABOUT_TO_START_CALLBACK_URL = StringUtils.join(
         "/", SOLICITOR_CREATE, ABOUT_TO_START_WEBHOOK
     );
 
     @Test
     public void shouldUpdateLanguagePreferenceSuccessfullyWhenAboutToStartCallbackIsInvoked()
```

Comment:
`shouldUpdateLanguagePreferenceAndPetitionerOrgDetailsSuccessfullyWhenAboutToStartCallbackIsInvoked`

Reply:
actually the org policy doesn't get set, it's set by default by CCD so the change I made to the test makes sure that when CCD sends it it comes back without being modified